pandas/parser.pyx

```diff
@@ -1750,6 +1772,78 @@ cdef inline int _try_double_nogil(parser_t *parser, int col,
 
     return 0
 
+cdef _try_uint64(parser_t *parser, int col, int line_start, int line_end,
+                 bint na_filter, kh_str_t *na_hashset):
```

Comment:
prob for another PR, but we should generate these try_* routines with tempita (if possible)